windows/win_service.ps1

```diff
@@ -0,0 +1,111 @@
+#!powershell
+#
+# WANT_JSON
+# POWERSHELL_COMMON
+
+
+$change=$false;
+$fail=$false;
+$msg=@();
+
+$params=Parse-Args $args;
+
+$name=Get-Attr $params "name" $false;
+$state=Get-Attr $params "state" "";
+$enabled=Get-Attr $params "enabled" "";
+
+if ($name -eq $false) {
+    $msg+=@("name is a required argument");
+}
+
+if (($state -eq "") -and ($enabled -eq "")) {
+    $msg+=@("you have to specify at least state or enabled");
+}
+
+if ($msg -ne @()) {
+    $result=New-Object psobject @{
+        changed=$false
+        failed=$true
+        msg=$msg
+    };
+    Exit-Json($result);
+}
+
+if ($state -ne "") {
+    if (($state -ne "running") -and ($state -ne "stopped") -and ($state -ne "paused")) {
+        $msg+=@("state only accepts the following arguments ['running', 'stopped', 'paused']");
+        $result=New-Object psobject @{
+            changed=$change
+            failed=$true
+            msg=$msg
+        };
+        Exit-Json($result);
+    } else {
+        $serviceState = (Get-Service $name).status;
+        if ($serviceState -ne $state) {
+            try {
+                Set-Service $name -Status $state;
+                $msg+=@("Service state set to $state");
+                $change=$true;
+            } catch [system.exception] {
+                $msg+=@("Failed to set service state.");
+                $result=New-Object psobject @{
+                    changed=$change
+                    failed=$true
+                    msg=$msg
+                };
+                Exit-Json($result);
+            };
+        } else {
+            $msg+=@("Service state was already set to $state");
+        };
+    };
+};
+
+
+if ($enabled -ne "") {
+    if (($enabled -eq $true) -or ($enabled -eq "true")) {
```

Comment:
The main reason I did add the Boolean option, was to keep it consistent with the Linux service module.
But if it would be better to lose that, I'll do that.